src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const list = document.querySelector('#thumbs');
+
+list.addEventListener('click', ev => {
```

Comment:
```suggestion
list.addEventListener('click', event => {
```

Reply:
![image](https://user-images.githubusercontent.com/110824803/199235515-9d652327-9a82-455f-8fbc-77637682ce6c.png)
I can't do it(